.cursor/commands/fix-build-test.md

```
@@ -0,0 +1,164 @@
+```
+
+If the workflow logs are available, review them to identify:
+- **Build failures**: TypeScript compilation errors from `tsc`
+- **Test failures**: Jest test failures from `npm run test:ci`
+
+### 2. Fix Build Errors First
+
+Build errors block test execution. Always fix these first.
+
+**Run the build:**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)